internal/venice-client-common/src/main/java/com/linkedin/venice/stats/dimensions/HttpResponseStatusCodeCategory.java

```diff
@@ -8,7 +8,7 @@
  * Maps the provided HTTP response status {@link HttpResponseStatus} to one of
  * 1xx, 2xx, 3xx, 4xx, 5xx categories.
  */
-public class VeniceHttpResponseStatusCodeCategory {
+public class HttpResponseStatusCodeCategory {
```

Comment:
This is just a utility method, but kept it as a class separately to be uniform like other dimensions. What is the need of a constructor here?

Reply:
Utility classes with only static methods are not intended to be constructed, they are only meant to be used statically, so we can add a private constructor to prevent them from being constructed.

Reply:
Added this, thanks.